### ISSUE TITLE
Loss = NaN

### ISSUE THREAD

**Issue by nitinprakash96** [nitinprakash96/RNN-language-model#2]
Receiving loss = NaN when activation is set to Sigmoid. Possible reasons:
- Gradient blow up
- input contains nan (or unexpected values) 
- Loss function
